drake/examples/Acrobot/test/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if (LCM_FOUND)
```

Comment:
I doubt this explains your MSVC problem, but you probably want `AND GTEST_FOUND` here